bcipy/helpers/tests/test_convert.py

```diff
@@ -0,0 +1,113 @@
+"""Tests for data conversion related functionality."""
+import os
+import random
+import shutil
+import tempfile
+import unittest
+import warnings
+from itertools import chain
+from pathlib import Path
+from typing import List
+
+import numpy as np
+from mne.io import read_raw_edf
+
+from bcipy.helpers.convert import convert_to_edf
+from bcipy.helpers.parameters import Parameters
+
+
+def sample_data(rows: int = 1000, ch_names: List[str] = ['c1', 'c2',
```

Comment:
I wonder if there's a better place for this function to live - acquisition? Otherwise, let's pull in the generate random data function to populate the channel_data

Reply:
The generation of sample data should definitely come from the acquisition generators. I'll update the code here.

As far as reading and writing a raw_data file, I think we're missing an abstraction/module. This logic is repeated a number of places throughout the code. I will create a followup ticket to clean this up since it affects a number of places.